README.md

```diff
@@ -1 +1,43 @@
-# Hello-Microverse
\ No newline at end of file
+Project Name
+
+    Hello Microverse
+
+Built With
+
+    HTML
+    CSS
+
+Usage
+
+This is my first project in Microverse program.This project aim at practicing the below :
+
+1)Linters
+2)git and github professional best practices
+
+
+👤 Author
+
+    GitHub: @massaillon
+    LinkedIn: Massaillon Silue
+    
+🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+Show your support
+
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+
+    Hat tip to anyone whose code was used
+    Inspiration
```

Comment:
- [ ]  OPTIONAL
 Great job on implementing the changes required by the previous reviewer. Below are some things to bear in mind and consider as **MANDATORY REQUIREMENT** in your next project.
 - Your PR title should be very descriptive, instead of `Added index to the repository`, you could use `Hello Microverse, greetings`.
 - Your readme authors should link to your `GitHub, LinkedIn, and Twitter account`. Do try maintaining consistency in following this [microverse readme template](https://github.com/microverseinc/readme-template)
 - Your `text.txt` should not be showing in your repo.
 - Your PR should be more descriptive pointing out what you did and summarizing all you did.

Reply:
HI sir
Noted OK 
I will reconsider your suggestions
thanks for the strong support